packages/components/src/utils/accessibility.ts

```diff
@@ -1,2 +1,16 @@
+import { StringOrTextWithA11y } from '../types/common'
+
 // Export related hooks
 export { useIsScreenReaderEnabled } from './hooks/useIsScreenReaderEnabled'
+
+/**
+ * Returns text that should be displayed on the screen
+ */
+export const getDisplayText = (text: StringOrTextWithA11y): string =>
```

Comment:
Not a hook and exported:
```suggestion
export const GetDisplayText = (text: StringOrTextWithA11y): string =>
```